sentence_transformers/sparse_encoder/evaluation/SparseNanoBEIREvaluator.py

```diff
@@ -189,4 +190,6 @@ def __call__(
     def _load_dataset(
         self, dataset_name: DatasetNameType, **ir_evaluator_kwargs
     ) -> SparseInformationRetrievalEvaluator:
+        ir_evaluator_kwargs["max_active_dims"] = self.max_active_dims
+        ir_evaluator_kwargs.pop("truncate_dim")
```

Comment:
```suggestion
        ir_evaluator_kwargs.pop("truncate_dim", None)
```
Just a bit safer in case we ever change the `InformationRetrievalEvaluator` or something